x-pack/plugins/enterprise_search/public/applications/app_search/components/crawler/components/add_domain/add_domain_flyout.tsx

```diff
@@ -0,0 +1,101 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React, { useState } from 'react';
+
+import {
+  EuiButton,
+  EuiButtonEmpty,
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiFlyout,
+  EuiFlyoutBody,
+  EuiFlyoutFooter,
+  EuiFlyoutHeader,
+  EuiPortal,
+  EuiSpacer,
+  EuiText,
+  EuiTitle,
+} from '@elastic/eui';
+
+import { i18n } from '@kbn/i18n';
+
+import { CANCEL_BUTTON_LABEL } from '../../../../../shared/constants';
+
+import { AddDomainForm } from './add_domain_form';
+import { AddDomainFormErrors } from './add_domain_form_errors';
+import { AddDomainFormSubmitButton } from './add_domain_form_submit_button';
+
+export const AddDomainFlyout: React.FC = () => {
+  const [isFlyoutVisible, setIsFlyoutVisible] = useState(false);
+
+  return (
+    <>
+      <EuiButton
+        size="s"
+        color="secondary"
```

Comment:
https://elastic.github.io/eui/#/navigation/button

> **secondary color is DEPRECATED, use success instead**

```suggestion
        color="success"
```

Reply:
I've replaced this in 3a32439